android-iconify-fontawesome/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.1.0'
```

Comment:
These were left in here. Luckily the upgrade grade PR merged. Rebase from master to resolve this.